tests/microbenchmarks.py

```diff
@@ -4,6 +4,7 @@
 
 from utils import *
```

Comment:
At column 1: (F403) 'from utils import *' used; unable to detect undefined names